Sources/Testing/Events/Recorder/Event.ConsoleOutputRecorder.swift

```diff
@@ -76,6 +76,12 @@ extension Event {
       /// This option is ignored unless ``useANSIEscapeCodes`` is also
       /// specified.
       case useTagColors(_ tagColors: [Tag: Tag.Color])
+
+      /// Record verbose output.
+      ///
+      /// When specified, additional output is provided. The exact nature of the
+      /// additional output is implementation-defined and subject to change.
+      case recordVerboseOutput
```

Comment:
The other options here have "use" as their leading word, so I'm wondering why this one uses "record". They all affect the behavior of the recording action

Reply:
I couldn't come up with better phrasing. Any ideas?

Reply:
`useVerboseOutput`

Reply:
Sure, will change.

Reply:
(We could also probably just drop `use` from all the cases as redundant.)